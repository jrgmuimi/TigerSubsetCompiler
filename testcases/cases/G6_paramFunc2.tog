let

    var p1:int
    var p2:int
    function Pfunc1(p1:int):void = 
        printstring("I got a param! - ");
        printint(p1)
    end

    function Pfunc2(p1:int, p2:int):void = 
        printstring("I got two params! - ");
        printint(p1);
        printstring(" - ");
        printint(p2)
    end
in
    p1:=0;
    p2:=9;
    Pfunc1(p2);
    printstring("\n");
    Pfunc2(p2,p1)

end