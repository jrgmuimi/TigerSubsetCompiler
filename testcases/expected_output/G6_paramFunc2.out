MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

I got a param! - 9
I got two params! - 9 - 0
